### PR TITLE
Preparing changelog for 0.10.1 release

### PR DESCRIPTION
**Pull request by denisonbarbosa**
DEENG-608

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,24 @@
+adsys (0.10.1) lunar; urgency=medium
+
+  [ Denison Barbosa ]
+  [ Jean-Baptiste Lallement ]
+  [ Gabriel Nagy ]
+  [ Didier Roche ]
+  * Fix erroneous non alternative dependency on package krb5-user
+  * Fix a bug in internal/config tests that was causing the autopkgtests to fail
+  * Update internal/config to also trigger a reload when config file is
+    overwritten
```
Comment (denisonbarbosa) [inline debian/changelog:10]: Was I to vague or to specific? I have no idea if I should describe more of the problem

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,24 @@
+adsys (0.10.1) lunar; urgency=medium
+
+  [ Denison Barbosa ]
+  [ Jean-Baptiste Lallement ]
+  [ Gabriel Nagy ]
```
[the remark this answers is not in the document]
Comment (denisonbarbosa) [reply debian/changelog:5]: This is a team effort, we all deserve the credit (and the blame if something goes wrong :joy: )

File: debian/changelog (hunk 1 above)
Comment (CarlosNihelton) [reply debian/changelog:10]: `git blame` disagrees 🤣

File: debian/changelog (hunk 2 above)
[the remark this answers is not in the document]
Comment (CarlosNihelton) [reply debian/changelog:5]: > This is a team effort, we all deserve the credit (and the blame if something goes wrong joy )

`git blame` disagrees 🤣

[the remark this answers is not in the document]
Comment (didrocks) [reply debian/changelog:5]: Can I opt-in for the credit and opt-out for the blame? :)

**Review by didrocks — APPROVED** [review]
Ah, I can approve now :)

**Pull request merged**